### ISSUE TITLE
[MAGO] Talento Impacto

### ISSUE THREAD

**Issue by Zangetsuha** [RoDNet/Bugtracker-Infinium-4.3.4#4631]
El talento da a los hechizos dañinos un 10% de probabilidad de reiniciar el cd del poder explosion de fuego, de hacer que explosion de fuego aturda al enemigo por 2 seg y que propague todos los daños periodicos que tiene el objetivo de la explosion de fuego...
http://imgur.com/a/4uCCW (LAS FOTOS ESTAN EN ORDEN CONTRARIO, LA ULTIMA ES LA PRIMERA, LA SEGUNDA ES LA PENULTIMA Y VA EN ESE ORDEN)
El problema es que ya no propaga los daños periodicos (en la foto se usa bomba viva de prueba) a los enemigos cercanos... Es un bug muy importante para el mago tanto PvE y PvP....

**Comment by Bichoculiao** [comment]
exacto, ademas que dicen que arreglaron piromano  Spells/Mago: Arreglado talento piromano, ahora saltará cuando bomba viva afecte a 3 objetivos a la vez y resulta ser que esa habilidad no funciona SOLO con bomba viva, sino con cualquier dot sobre 3 o mas enemigos... cada vez aparecen mas bugs que ya se encontraban arreglados en los magos... -.-

**Comment by jimmykell** [comment]
Efectivamente, está comprobado que no funciona Pirómano e Impacto no propaga los efectos dañinos en el tiempo a los mops cercanos. Esto antes funcionaba perfectamente. Como bien decis es una rotación del mago fundamental para pulls en raids, que además nos hace subir la celeridad y por tanto el DPS.

**Comment by efectokoleos** [comment]
deberia estar arreglado

**Comment by jimmykell** [comment]
Ahora si propaga los efectos dañinos en el tiempo, pero NO funciona Pirómano. 
Si funciona Pirómano, cuando afectas a más de 3 mops con fogonazo. Esta mal ya que no debería saltar pirómano con fogonazo y si con daños en el tiempo.
